assets/app/scripts/controllers/catalog/images.js

```diff
@@ -25,7 +25,6 @@ angular.module('openshiftConsole')
               imageRepoTag: imageRepoTag,
               name: imageRepo.metadata.name + ":" + imageRepoTag
             };
-            $scope.images.push(image);
```

Comment:
The longer term, probably better solution would be to get each stream, and for each tag in status.tags, get the related ImageStreamTag, which one day should include any labels (such as builder) converted to annotations. But that's not there now.